cloudrds/cloudrds.go

```diff
@@ -0,0 +1,144 @@
+package cloudrds
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"sync"
+
+	// we need mysql import because it's called in file, (but compile will complain because there is no display)
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/iotexproject/iotex-core/pkg/lifecycle"
+)
+
+var (
+	awsRDSEndpoint = "iotex-explorer-db.ctcedgqcwrb5.us-west-1.rds.amazonaws.com"
+	awsRDSPort     = 4086
+	awsRegion      = "us-west-1"
+	awsRDSUser     = "explorer_admin"
+	awsPass        = "j1cDiH7W7QCB"
+	awsDBName      = "explorer"
+)
+
+// RDSStore is the interface of KV store.
+type RDSStore interface {
```

Comment:
It seems to me that the abstraction is a little trivial. I wonder if we really need these interface, or we should call rds directly.

Reply:
I want to hide the detail of initial client , parse results and transaction